db/migrate/20200710101438_add_non_uk_gcse_columnns_to_application_qualification.rb

```diff
@@ -0,0 +1,8 @@
+class AddNonUkGcseColumnnsToApplicationQualification < ActiveRecord::Migration[6.0]
+  def change
+    add_column :application_qualifications, :non_uk_qualification_type, :string
+    add_column :application_qualifications, :accrediting_country, :string
```

Comment:
I'm not convinced about this, but can't think of anything better. Maybe awarding_country?

Reply:
I prefer `awarding_country` 👍

Reply:
For international degrees we've got `institution_country` but I think that's a different thing.